### DISCUSSION TITLE
Blue Iris produced invalid deepstack files on NAS

### DISCUSSION THREAD

**Discussion by mikerenna** [neilenns/node-deepstackai-trigger#462] [General]
the files output from Blue Iris to a NAS share (same NAS running the containers), i get an error from deepstack that No valid file found. However, if i drag the same file out of that folder and back in using windows shares, it processes the file properly. Something about the way blue iris writes the file across the network to the NAS share leaves it unreadable by deepstack. I've tried writing locally and then sharing the Blue Iris folder but struggled with how reference that in docker compose. I tried mounting that same folder within my qnap sofware (using hybridmount) to make it easier to figure out references in docker compose but no luck.

Anyone know why the file lands invalid and how to correct? Or how to reference other network shares from a docker compose file?

**Comment by neilenns** [comment]
I don't, sorry. I do know that other people have struggled with network shares being involved. It is just really messy when you have network shares, Docker, and other Docker management tools (e.g. Portainer) involved.

Is there any way you can get BlueIris and the Docker container running this on the same machine? That's how I have it set up.

**Reply by mikerenna** [reply]
That seems like a good idea. What do you do to run docker? Since my blue iris machine is windows and docker desktop doesn't look like it runs without a user being logged in.

**Reply by neilenns** [reply]
I'm just using Docker Desktop on Windows with WSL2 installed. I leave the machine logged in with a local user if I remember right, and somehow configured Windows to auto-login that user after reboots. Works fine.
